rozy/src/utils.rs

```diff
@@ -9,3 +10,19 @@ pub fn download_to(dest_path: &std::path::PathBuf, url: &str) -> Result<()> {
     std::fs::rename(tmp_dest_path, dest_path)?;
     Ok(())
 }
+
+pub fn run_with_stderr_to_stdout(
+    mut command: std::process::Command,
+) -> Result<std::process::ExitStatus> {
+    let mut spawned = command.spawn().unwrap();
+
+    let stdout = spawned.stdout.as_mut().unwrap();
+    let stdout_reader = BufReader::new(stdout);
+    let stdout_lines = stdout_reader.lines();
+
+    for line in stdout_lines {
```

Comment:
If I'm reading this correctly: this doens't pipe stdout to stderr, it buffers stdout and then reads it one line at a time and re-outputs.

Is there no way to control the child process's file descriptors? In regular C style code or python equivalent, you'd redirect the child's stdout to your own stderr, and then the OS just does the right thing.

Reply:
Yeah, I'd copied this code from here: https://stackoverflow.com/a/50444713. Specifying file descriptors is certainly less circuitous and requires no buffering.